src/pages/home/report/ReportActionItem.js

```diff
@@ -687,6 +695,7 @@ export default compose(
             key: ({action}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS_REACTIONS}${action.reportActionID}`,
         },
     }),
+    withRoute,
```

Comment:
As `ReportActionItem` is function component, why cannot we use `useRoute` hook directly?

Reply:
Yeah, you're right. We are moving away from HOCs and transitioning to functional components. Thanks for pointing it out. I'll update it shortly